lib/minion.ex

```diff
@@ -84,4 +98,21 @@ defmodule AssembleTheMinions.Minion do
     {:noreply, new_state}
   end
 
+  # count basic arithmetic
+  def handle_cast({:add, n}, state) when is_integer(n) and n >= 0 do
```

Comment:
Guards here are probably overkill as you're handling them in the plain function calls